src/api/procedures/__tests__/payDividends.ts

```diff
@@ -170,6 +170,8 @@ describe('payDividends procedure', () => {
     const dids = ['someDid', 'otherDid'];
     const targets = [dids[0], entityMockUtils.getIdentityInstance({ did: dids[1] })];
 
+    sinon.stub(utilsConversionModule, 'boolToBoolean').returns(true);
```

Comment:
Why is this necessary?

Reply:
`should throw an error if some of the supplied targets has already claimed their benefits` brokes if I remove that stub.

Reply:
Then you're not returning the proper values in the `holderPaid` stub